components/logins/src/migrate_sqlcipher_db.rs

```diff
@@ -19,19 +55,18 @@ pub fn migrate_sqlcipher_db_to_plaintext(
     salt: Option<&str>,
 ) -> Result<MigrationMetrics> {
     let mut db = Connection::open(old_db_path)?;
-    let metrics = init_sqlcipher_db(&mut db, old_encryption_key, salt, new_encryption_key)?;
-    sqlcipher_export(&mut db, &new_db_path)?;
-    let new_db = Connection::open(new_db_path)?;
-    new_db.execute("PRAGMA user_version = 1", NO_PARAMS)?;
+    init_sqlcipher_db(&mut db, old_encryption_key, salt)?;
+
+    // Init the new plaintext db as we would a regular client
+    let plaintext_store = LoginStore::new(new_db_path)?;
```

Comment:
nit: I would suggest renaming 'plaintext_store'. It sounds like we're storing unencrypted login records.